src/core/schedule.js

```diff
@@ -67,3 +43,35 @@ export const getFullSchedule = shiftWeek => {
 		})
 	return scheduleWeek.join('')
 }
+
+const getDayIndex = () => {
+	return {
+		today() {
+			const date = new Date()
+			const currentHours = date.getUTCHours()
+			// Time zone offset in Moscow
+			date.setUTCHours(currentHours + 3)
+			const currentDay = date.getDay()
+			const dayIndex = currentDay % 7
+			return dayIndex % 7
+		},
+		tomorrow() {
+			return this.today() + 1
+		},
+	}
+}
```

Comment:
1. This function name doesn't address its return result. I expect it to return an index (number) 
2. Why not add two separate functions `getTodayIndex` and `getTomorrowIndex`? Current version looks too much complicated

Reply:
1.This function returns the day index
Here is the result of execution
`console.log(getDayIndex().today(new Date()))
`
![image](https://github.com/mkangoo/Telegram-bot-schedule/assets/121447049/102bbd06-f8c4-415b-81d6-22548f331742)
2.I came up with the idea of ​​doing something like `try().expect()` in jest.
I'll break these methods into two functions, just like you said.